tests/test_wave_01.py

```diff
@@ -130,14 +134,16 @@ def test_update_task_not_found(client):
 
     # Assert
     assert response.status_code == 404
+    assert "message" in response_body
```

Comment:
Same comment as above. How can you write this assertion so that we're not just looking for "message" but we're looking at the entire response_body?